controllers/machineset_controller.go

```diff
@@ -230,15 +233,11 @@ func (r *MachineSetReconciler) reconcile(ctx context.Context, cluster *clusterv1
 	for _, machine := range filteredMachines {
 		if conditions.IsFalse(machine, clusterv1.MachineOwnerRemediatedCondition) {
 			log.Info("Deleting unhealthy machine", "machine", machine.GetName())
-			patch := client.MergeFrom(machine.DeepCopy())
 			if err := r.Client.Delete(ctx, machine); err != nil {
 				errs = append(errs, errors.Wrap(err, "failed to delete"))
 				continue
 			}
 			conditions.MarkTrue(machine, clusterv1.MachineOwnerRemediatedCondition)
-			if err := r.Client.Status().Patch(ctx, machine, patch); err != nil && !apierrors.IsNotFound(err) {
-				errs = append(errs, errors.Wrap(err, "failed to update status"))
-			}
```

Comment:
By removing this Machine patch, the `MachineOwnerRemediatedCondition` won't be set, unless this is done somewhere else?